client/question/components/PrivacyModalDirective.js

```diff
@@ -0,0 +1,159 @@
+            </p>
+          <div class="tie-privacy-modal-button-container">
+          <button ng-show="isClientVersion()" class="tie-button-blue tie-privacy-modal-button" ng-click="closeModal()">
+              <span>Okay</span>
+            </button>
+            <button ng-show="!isClientVersion()" class="tie-button-blue tie-privacy-modal-button" ng-click="accept()">
+              <span>Accept</span>
+            </button>
+            <button ng-show="!isClientVersion()" class="tie-button-red tie-privacy-modal-button" ng-click="reject()">
+              <span>Reject</span>
+            </button>
+          </div>
+          <div ng-show="!isClientVersion()">
+            <label>
+              <input type="checkbox" ng-model="modalData.isUsingSharedComputer"></input>
+              This is a shared computer, don't save my response.
+            </label>
+          </div>
+        </div>
+      </div>
+      <style>
+        .tie-privacy-modal {
+          background-color: rgb(0, 0, 0);
+          background-color: rgba(0, 0, 0, 0.4);
+          height: 100%;
+          left: 0;
+          overflow: auto;
+          position: fixed;
+          top: 0;
+          width: 100%;
+          z-index: 4;
+        }
+        .tie-privacy-modal-button {
+          border-radius: 4px;
+          border-style: none;
+          cursor: pointer;
+          font-family: Roboto, 'Helvetica Neue', 'Lucida Grande', sans-serif;
+          font-size: 14px;
+          height: 50px;
+          padding: 1px 6px;
+          width: 180px;
+        }
+        .tie-privacy-modal-button-container {
+          display: flex;
+          justify-content: space-around;
+          padding: 15px 0px;
+        }
+        .tie-privacy-modal-content {
+          background-color: white;
+          border: 1px solid #969696;
+          border-radius: 10px;
+          box-shadow: 0px 5px 20px 2px #797979;
+          margin: 15% auto;
+          padding: 20px;
+          text-align: center;
+          width: 30%;
+          z-index: 5;
+        }
+        .tie-privacy-modal-content p {
+          font-size: 16px;
+          color: #7b7b7b;
+        }
+        .tie-privacy-modal-title {
+          font-size: 24px;
+          font-weight: 600;
+        }
+      </style>
+    `,
+    controller: ['$cookies', '$scope', '$window', 'PRIVACY_COOKIE_LIFETIME',
+      'PRIVACY_COOKIE', 'MENU_PAGE_URL_FROM_QUESTION',
+      function($cookies, $scope, $window, PRIVACY_COOKIE_LIFETIME,
+        PRIVACY_COOKIE, MENU_PAGE_URL_FROM_QUESTION) {
+        // Need to create this to be able to use ng-model, see below
+        // https://stackoverflow.com/questions/12618342/ng-model-does-not-update-controller-value/22768720#22768720
+        $scope.modalData = {};
+
+        /**
+         * Stores the user's acceptance of the privacy policy
+         * and closes the modal.
+         */
+        $scope.accept = function() {
+          // If this is not a shared computer
```

Comment:
Don't need this comment; it just repeats the next line of code in words. Comments are better used to explain _why_ something is the case.